runelite-client/src/main/java/net/runelite/client/plugins/devtools/DevToolsPlugin.java

```diff
@@ -283,35 +282,55 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 			return;
 		}
 
-		MenuAction action = MenuAction.of(event.getType());
+		final ExamineType examineType = ExamineType.of(MenuAction.of(event.getType()));
```

Comment:
Maybe instead of doing what are you ding at line 297 you should instead check for last enu entry and not check for w/e is in MenuEntryAdded event

Reply:
I've divested from the event as much as possible here. (It is now only used to guard against adding extra "ID: " suffixes to existing examine entries)